backend/lobby/consumers.py

```diff
@@ -1,8 +1,13 @@
-from channels.generic.websocket import WebsocketConsumer
 import json
 
+from channels.generic.websocket import WebsocketConsumer
+
+from backend.game.engine.board import BoardGame
+
 
 class GameConsumer(WebsocketConsumer):
+    master_game_controller = BoardGame()
```

Comment:
The channels changes made within branch 'add_channel_roll_cmd' will have you accessing the latest BoardGame instance for the room via DB...

We do not have a PR for this yet. Once we do or its merged I can let you know.